pages/community/docs/governance/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: 'Governance'
+description: 'Because AsyncAPI is a large, complex tool that touches so much of the IT ecosystem, governance policies are important to meet both internal and external standards.'
```

Comment:
```suggestion
description: 'Because AsyncAPI is a large, complex tool that touches so much of the IT ecosystem, governance policies are important to meet internal and external standards.'
```